sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/QueryPlanRetriever.java

```diff
@@ -43,7 +44,20 @@ class QueryPlanRetriever {
                                                                QueryFeature.GroupBy.name() + ", " +
                                                                QueryFeature.Top.name() + ", " +
                                                                QueryFeature.DCount.name() + ", " +
-                                                               QueryFeature.NonValueAggregate.name();
+                                                               QueryFeature.NonValueAggregate.name() + ", " +
+                                                               QueryFeature.NonStreamingOrderBy.name();
+
+    private static final String OLD_SUPPORTED_QUERY_FEATURES = QueryFeature.Aggregate.name() + ", " +
```

Comment:
can we add some comments about why maintain the old_query_feature vs new_query_feature?

Reply:
Added necessary documentation.